### PR TITLE
[PR] add install instructions

### PR DESCRIPTION
**Pull request by divramod**
install is broken, i added a simple workaround to the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,12 +11,63 @@
...
+#!/usr/bin/env argsh
+set -eu -o pipefail
+
+# --- OPTIONS
+# argsh(t|test-option): env(TEST_OPTION) def(false) des(A test option.) val()
+
+# --- ACTIONS
+# argsh(T|test-action):   des(A test action.)
+
+# --- ACTIONS
```
Comment (fentas) [inline README.md]: could you remove this?

File: README.md [hunk 2]
```diff
@@ -11,12 +11,63 @@
...
+
+# --- PATHS
+: "${PATH_BASE:="$(git rev-parse --show-toplevel)"}"
```
Comment (fentas) [inline README.md]: could you remove this?
Not every script has to be within a git repo.

File: README.md [hunk 3]
```diff
@@ -11,12 +11,63 @@
...
+
+# --- VARIABLES
+
+# --- IMPORTS
+
+# --- MAIN
```
Comment (fentas) [inline README.md]: could you remove this? `# --- ...`

**Review by fentas — COMMENTED** [review]
Thanks for PR. Could you do some adjustments?
Mainly remove empty comments.

**Comment by divramod** [comment]
@fentas ready for merge :-)

**Comment by fentas** [comment]
you sneaked in the `/usr/bin/env awk -f` fix :smirk_cat: 
Thanks :+1:

**Pull request merged**